flink-tests/src/test/java/org/apache/flink/test/checkpointing/StatefulJobSnapshotMigrationITCase.java

```diff
@@ -136,6 +136,12 @@ public void testSavepoint() throws Exception {
         switch (snapshotSpec.getStateBackendType()) {
             case StateBackendLoader.ROCKSDB_STATE_BACKEND_NAME:
                 env.setStateBackend(new EmbeddedRocksDBStateBackend());
+
+                if (executionMode == ExecutionMode.CREATE_SNAPSHOT) {
```

Comment:
@rkhachatryan That means that we don't have to create backports but only merge this PR (because for 1.17 I have to disable the changelog backend manually based on `release-1.17.0` and for 1.16 we have the migration test data already generated?!

Reply:
@XComp I think you're right, backporting isn't necessary.